cmd/common_test.go

```diff
@@ -103,6 +103,16 @@ func testCreateVolume(t *testing.T, volName string, size uint64) {
 	assert.True(t, util.ListContainsSubString(lines, fmt.Sprintf("Volume %s created with id", volName)))
 }
 
+// Takes a volume name and size. Returns the created volume id.
+func testCreateVolumeWithLabel(t *testing.T, volName string, size uint64, labels string) {
+	cli := "px create volume " + volName + " --size " + strconv.FormatUint(size, 10) +
```

Comment:
use fmt.Sprintf() instead. This is hard to manage with so many `+`

Please do this for all calls like in line 125

Reply:
Fixed for all other cases along with this.